### PR TITLE
fix(artifactPipeline): move collect_logs to working call

### LINKED ISSUE #3988: artifact failed on `SCT collect logs data()`
Seen for multiple SCT artifacts tests:
https://jenkins.scylladb.com/view/master/job/scylla-master/job/artifacts/job/artifacts-ubuntu2004-test/323/
and also in  https://jenkins.scylladb.com/view/master/job/scylla-master/job/artifacts/job/artifacts-amazon2-test/287/

from ubuntu:20.04 logs i see the following error
```
11:59:01  < t:2021-09-30 08:58:51,290 f:wait.py         l:65   c:sdcm.wait            p:ERROR > Wait for: Checking if package manager is free: timeout - 60 seconds - expired
11:59:01  < t:2021-09-30 08:58:51,290 f:wait.py         l:69   c:sdcm.wait            p:ERROR > last error: RetryError(<Future at 0x7ff611ebd9d0 state=finished returned bool>)
```

and 
```12:07:51  < t:2021-09-30 09:07:51,703 f:threads_and_processes_alive.py l:53   c:sdcm.utils.threads_and_processes_alive p:ERROR > There are some threads left alive from following modules: multiprocessing.queues,sdcm.remote.remote_cmd_runner,paramiko.transport
12:07:51  < t:2021-09-30 09:07:51,703 f:tester.py       l:2321 c:ArtifactsTest        p:ERROR > Please check /jenkins/slave/workspace/scylla-master/artifacts/artifacts-ubuntu2004-test/scylla-cluster-tests/20210930-085314-419721/left_processes.log log to see them
```

### PR DESCRIPTION
**Pull request by fgelcer**
since 0b1a2f8190272196e8364a4c8267fbfc08c32db0 the collect_logs
function seems broken, so moving it back to a hydra
cli call to have them fixed with a better commit, or
to revert a broken commit.
fixed #3988

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Review by yaronkaikov — APPROVED** [review]

**Comment by yaronkaikov** [comment]
@fgelcer Thanks, i see that @amoskong also sent https://github.com/scylladb/scylla-cluster-tests/pull/3993

File: vars/artifactsPipeline.groovy [hunk 1]
```diff
@@ -189,13 +189,12 @@ def call(Map pipelineParams) {
                                                 """
                                             }
                                             stage("Collect log data (${instance_type})") {
-                                                catchError(stageResult: 'FAILURE') {
-                                                    wrap([$class: 'BuildUser']) {
-                                                        dir('scylla-cluster-tests') {
-                                                            runCollectLogs(params, builder.region)
```
Comment (amoskong) [inline vars/artifactsPipeline.groovy:195]: The is correct, but the same problem exists in runCleanupResource(), runSendEmail().

Comment (fgelcer) [reply vars/artifactsPipeline.groovy:195]: i have reverted it for all these functions as well @amoskong

**Comment by bentsi** [comment]
no need to run `test_provions_*` it doesn't check artifacts pipeline

**Comment by fgelcer** [comment]
> no need to run `test_provions_*`, it doesn't check the artifacts pipeline

ok

File: vars/artifactsPipeline.groovy (hunk 1 above)
Comment (amoskong) [reply vars/artifactsPipeline.groovy:195]: OK, I'm ok with this QUICK fix if we want to make the jobs back to stable ASAP.

**Review by amoskong — APPROVED** [review]

**Comment by bentsi** [comment]
this is an incorrect solution... need to clean SCT Runner IP file. Closing this since https://github.com/scylladb/scylla-cluster-tests/pull/3993 should solve this

**Pull request closed without merging**